pages/common/latexmk.md

```diff
@@ -15,9 +15,9 @@
 
 `latexmk -pdf {{source.tex}}`
 
-- Clean up all temporary tex files in the folder:
+- Force the generation of a document inspite of errors:
```

Comment:
The correct spelling here would be "in spite of errors", but I think it would be clearer as "even if there are errors" or "disregarding any errors".